yarn.lock

```diff
@@ -2645,7 +2645,7 @@ ecc-jsbn@~0.1.1:
 
 "eclipse-che-operator@git://github.com/eclipse-che/che-operator#main":
   version "0.0.0"
-  resolved "git://github.com/eclipse-che/che-operator#d52d5adb16e751699f257561836b3582cd6e26f9"
+  resolved "git://github.com/eclipse-che/che-operator#b8cc02bf9a6a0d6f4eb5d64e02feae8b51653bcd"
```

Comment:
I think you don't relay on operator resources in this PR, so this change will only cause merge conflicts.